diff -up sidplay-libs-2.1.1/builders/hardsid-builder/configure.ac~ sidplay-libs-2.1.1/builders/hardsid-builder/configure.ac
--- sidplay-libs-2.1.1/builders/hardsid-builder/configure.ac~	2004-06-14 22:07:57.000000000 +0200
+++ sidplay-libs-2.1.1/builders/hardsid-builder/configure.ac	2011-09-08 09:35:29.319418972 +0200
@@ -19,7 +19,6 @@ AC_PROG_CC
 AC_PROG_CXX
 
 dnl Initialize libtool.
-AC_DISABLE_SHARED
 AC_LIBTOOL_WIN32_DLL
 AC_PROG_LIBTOOL
 
diff -up sidplay-libs-2.1.1/builders/resid-builder/configure.ac~ sidplay-libs-2.1.1/builders/resid-builder/configure.ac
--- sidplay-libs-2.1.1/builders/resid-builder/configure.ac~	2004-06-14 22:07:58.000000000 +0200
+++ sidplay-libs-2.1.1/builders/resid-builder/configure.ac	2011-09-08 09:35:37.751418859 +0200
@@ -19,7 +19,6 @@ AC_PROG_CC
 AC_PROG_CXX
 
 dnl Initialize libtool.
-AC_DISABLE_SHARED
 AC_LIBTOOL_WIN32_DLL
 AC_PROG_LIBTOOL
 
diff -up sidplay-libs-2.1.1/resid/configure.in~ sidplay-libs-2.1.1/resid/configure.in
--- sidplay-libs-2.1.1/resid/configure.in~	2004-06-14 22:08:23.000000000 +0200
+++ sidplay-libs-2.1.1/resid/configure.in	2011-09-08 10:21:16.644382190 +0200
@@ -71,7 +71,6 @@ AC_SUBST(RESID_HAVE_BOOL)
 dnl Checks for library functions.
 
 dnl Libtool
-AC_DISABLE_SHARED
 AM_PROG_LIBTOOL
 AC_SUBST(LIBTOOL_DEPS)
 AC_SUBST(LTVERSION)
diff -up sidplay-libs-2.1.1/libsidplay/unix/libsidplay2.pc.in~ sidplay-libs-2.1.1/libsidplay/unix/libsidplay2.pc.in
--- sidplay-libs-2.1.1/libsidplay/unix/libsidplay2.pc.in~	2004-06-14 22:08:04.000000000 +0200
+++ sidplay-libs-2.1.1/libsidplay/unix/libsidplay2.pc.in	2011-09-08 11:43:55.681315726 +0200
@@ -7,5 +7,5 @@ builders=@builders@
 Name: @PACKAGE@
 Description: C64 cycle exact stereo sidplayer library.
 Version: @VERSION@
-Libs: ${libdir}/libsidplay2.la
+Libs: -L${libdir} -lsidplay2
 Cflags: -DHAVE_UNIX -I${includedir}
diff -up sidplay-libs-2.1.1/libsidutils/unix/libsidutils.pc.in~ sidplay-libs-2.1.1/libsidutils/unix/libsidutils.pc.in
--- sidplay-libs-2.1.1/libsidutils/unix/libsidutils.pc.in~	2004-06-14 22:08:09.000000000 +0200
+++ sidplay-libs-2.1.1/libsidutils/unix/libsidutils.pc.in	2011-09-08 11:43:57.552315695 +0200
@@ -6,5 +6,5 @@ includedir=@includedir@
 Name: @PACKAGE@
 Description: Utils for supporting the libsidplay2 library.
 Version: @VERSION@
-Libs: ${libdir}/libsidutils.la
+Libs: -L${libdir} -lsidutils
 Cflags: -DHAVE_UNIX -I${includedir}
